packages/squiggle-lang/src/value/index.ts

```diff
@@ -135,17 +178,46 @@ export type LabeledDistribution = {
   distribution: BaseDist;
 };
 
-export type Plot = {
-  distributions: LabeledDistribution[];
-};
-
-class VPlot {
+export type Plot =
+  | {
+      type: "distributions";
+      distributions: LabeledDistribution[];
+    }
+  | {
+      type: "fn";
+      fn: Lambda;
+      min: number;
```

Comment:
Tiny point, but maybe min/max should be optional? Min=0, max=10.

Reply:
I considered this when I wrote the code, but right now we don't have the support in `frRecord` for optional fields. It could be done with duplicating the `Plot.fn` definition in all combinations, but I don't think it's worth the hassle.

I agree that it needs to be implemented if `Plot.fn` gets more arguments (color, etc).

Reply:
Makes sense, let's ignore for now.